MapboxNavigation.podspec

```diff
@@ -44,9 +44,7 @@ Pod::Spec.new do |s|
   s.module_name = "MapboxNavigation"
 
   s.dependency "MapboxCoreNavigation", "#{s.version.to_s}"
-  s.dependency "MapboxDirections.swift", "~> 0.10.0"
   s.dependency "Mapbox-iOS-SDK", "~> 3.5"
-  s.dependency "OSRMTextInstructions", "~> 0.2.0"
```

Comment:
Ah, I see, that’s because MapboxNavigation-Documentation.podspec [produces a single combined framework](https://github.com/mapbox/mapbox-navigation-ios/blob/c113e518aa38274cf67d64d95c4cb638e732c0a7/MapboxNavigation-Documentation.podspec#L35).